### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by davidwengier**
@RikkiGibson I think this should fix the build, and is simpler than a revert. If it helps, feel free to use it, or https://github.com/dotnet/roslyn/pull/54451 whichever you think is best.

### CODE REVIEW

**Review by RikkiGibson — APPROVED** [review]

**Review by dibarbet — APPROVED** [review]

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by RikkiGibson** [comment]
Merging without waiting for CI to unblock main build

**Pull request merged**
